src/Compute/Compute/Strategies/ComputeRp/Images.json

```diff
@@ -0,0 +1,84 @@
+{
+    "Linux":  {
+                  "CentOS":  {
```

Comment:
NIT: Can we have the same spaces for each level so that each level aligns?

For example, the same number of spaces before "publisher".

Reply:
Same as above - this is how PowerShell outputs json, so if I change the formatting it will just be overwritten next time the file is generated.